File: back/src/main/java/br/com/Helppy/model/Comentario.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 * Click nbfs://nbhost/SystemFileSystem/Templates/Classes/Class.java to edit this template
 */
package br.com.Helppy.model;

import java.sql.Timestamp;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;
import lombok.ToString;
import lombok.experimental.SuperBuilder;

/**
 *
 * @author criad
 */
@Data
@Setter
@Getter
@ToString
@SuperBuilder
@AllArgsConstructor
@NoArgsConstructor
public class Comentario {
    protected int idComentario;
    protected String post;
    protected Timestamp data;
    protected int quantidadeDenuncias;
    protected int idCategoria;
    protected int usuario_idUsuario;
}

File: front/src/app/model/comentario.ts
export class Comentario {
    idComentario: number;
    post: string;
    data: Date;
    quantidadeDenuncias: number;
    idCategoria: number;
    usuario_idUsuario: number;


    constructor(){
        this.idComentario = 0;
        this.post = "";
        this.data = new Date();
        this.quantidadeDenuncias = 0;
        this.idCategoria = 0;
        this.usuario_idUsuario = 0;
    }






}
